code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -943,7 +943,7 @@ datum
 			fluid_r = 99
 			fluid_g = 32
 			fluid_b = 15
-			alch_strength = 0.15 //adding milk shouldn't quadruple the alcohol per volume
+			alch_strength = 0.2 //adding milk shouldn't quadruple the alcohol per volume
```

Comment:
This one's an increase, is that intentional?

Reply:
oh it was a decrease to .15 but then i increased it from that up to .2

Reply:
it was a decrease to .15 but then i upped it a bit back to .2